packages/block-editor/src/components/block-popover/README.md

```diff
@@ -0,0 +1,41 @@
+# BlockPopover and BlockPopoverInbetween
+
+These two components allows rendering editor UI by the block (in a popover) but outside the the canvas. This is important to avoid messing up with the style and layout of the block list.
```

Comment:
These two components allows -> These two components allow
but outside the the canvas -> but outside the canvas

Reply:
messing up with the style -> messing with the style